### PR TITLE
fix(ONYX-1075): scroll performance on SWA landing page

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-1075] 

### Description

This PR improves the scroll performance on the SWA landing page. This is thanks to using `Flatlist` instead of `ScrollView`. 

FlatList renders only the visible items, optimizing performance with large datasets. ScrollView renders all items at once, potentially causing performance issues on devices with limited processing power.

The difference in performance between the videos is not so big but it would be pretty noticeable on an old android device.
**Before**

https://github.com/artsy/eigen/assets/11945712/3dbfc205-8403-4344-bc0a-1c29cfaa76cf

**After**

https://github.com/artsy/eigen/assets/11945712/19c3ee7a-6fbe-49ad-8b99-3d5c68346430

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1075]: https://artsyproduct.atlassian.net/browse/ONYX-1075?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx [hunk 1]
```diff
@@ -120,42 +113,66 @@ export const SellWithArtsyHome: React.FC = () => {
...
-              <SellWithArtsyRecentlySold recentlySoldArtworks={recentlySoldArtworks} />
-            )}
-
-            <Testimonials />
-
-            <SpeakToTheTeam onInquiryPress={handleInquiryPress} />
-
-            <Footer />
-          </Join>
+  const data = compact([
```
Comment (MounirDhahri) [inline src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:116]: we use this pattern in some other screens as well where we have performance issues.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ace2d2646d9fc653097370f9d23ff33412147b5a
</p>

File: src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx [hunk 2]
```diff
@@ -120,42 +113,66 @@ export const SellWithArtsyHome: React.FC = () => {
...
+    {
+      key: "footer",
+      content: <Footer />,
+    },
+  ])
 
-          <Spacer y={2} />
-        </ScrollView>
+  return (
+    <Flex flex={1}>
```
Comment (gkartalis) [inline src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:164]: Can we bring back the `screen` component here to make sure the layout doesn't break anywhere?

We can always use either `<Screen.Flatlist />` or even `<Flatlist />`

**Review by gkartalis — DISMISSED** [review]
👏 awesome improvement! Left one comment only

File: src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx (hunk 2 above)
Comment (MounirDhahri) [reply src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:164]: Good note, I can do that 🙌  I tried to eliminate any non default components but I think it's okay to use it.

**Review by olerichter00 — APPROVED** [review]
Good to know that ScrollView can cause performance issues with a lot of content 💡

**Comment by MounirDhahri** [comment]
@olerichter00 @gkartalis I just noticed my before video wasn't uploaded, I just uploaded it again.

**Review by gkartalis — APPROVED** [review]
🏆

**Review by olerichter00 — APPROVED** [review]

**Comment by dariakoko** [comment]
Niiiiice! Thank you!

**Pull request merged**